frontend/src/app/core/services/notification.service.ts

```diff
@@ -0,0 +1,25 @@
+import { inject, Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Injectable({ providedIn: 'root' })
+export class NotificationService {
+    public constructor(private snackBar: MatSnackBar) {
+        this.snackBar = inject(MatSnackBar);
+    }
+
+    public showErrorMessage(error: string) {
+        this.snackBar.open(error, '', { duration: 2500, panelClass: 'error-snack-bar' });
```

Comment:
Why one of the parameters is an empty string?

Reply:
I should make methods with 2 parameters or use some global constant for empty string ?

Reply:
I should make methods with 2 parameters or just use some kind of global constant for empty string ?

Reply:
We have notification without button. In snackBar.open second parameter it's name for action. That's why empty string is used